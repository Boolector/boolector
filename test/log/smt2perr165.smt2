(declare-const a (_ BitVec 4294967296))
