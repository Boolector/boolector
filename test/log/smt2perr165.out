log/smt2perr165.smt2:1:28: invalid 32-bit integer '4294967296'
